build.gradle

```diff
@@ -70,8 +71,8 @@ subprojects {
         compile group: 'com.google.code.findbugs', name: 'jsr305', version: '3.0.0'
 
         // SLF4J
-        compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.21'
-        compile group: 'org.slf4j', name: 'slf4j-jdk14', version: '1.7.21'
+        compile group: 'org.slf4j', name: 'slf4j-api', version: SLF4J_VERSION
+        compile group: 'org.slf4j', name: 'slf4j-jdk14', version: project.SLF4J_VERSION
```

Comment:
Why the notation is different? I.e. `SLF4J_VERSION` vs. `project.SLF4J_VERSION`?